examples/supermassive-todomvc/src/components/Todo.tsx

```diff
@@ -2,6 +2,7 @@ import React from "react";
 
 import { useFragment } from "@graphitation/apollo-react-relay-duct-tape";
 import { graphql } from "@graphitation/graphql-js-tag";
+import { TodoFragment$key as TodoFragmentType } from "./__generated__/TodoFragment.graphql";
```

Comment:
I'd rather not rename as it's not really true that this _is_ the type of the fragment data itself, rather it's an opaque fragment reference prop.